interview-questions.md

```diff
@@ -7,53 +7,75 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is a block in Ruby?
 
-  Your answer:
+  Your answer: 
+  A snippet of code encapsulated in curly braces. 
```

Comment:
'do' 'end' for multiple statements and {} for single statement is the preferred styling